libmozdata/patchanalysis.py

```diff
@@ -563,6 +563,21 @@ def attachmenthandler(attachments, bugid):
         if info['uplift_comment'] and info['uplift_author']:
             break
 
+    # Landing dates per useful channels
```

Comment:
Could you move this near the other `landing_delta` code and avoid calling `Bugzilla.get_landing_comments`?